pages.es/common/grep.md

```diff
@@ -0,0 +1,37 @@
+# grep
```

Comment:
I think you translate an old version of the `grep` page, if you use a client that supports cache, like the python client you should use `tldr -u` to update the local cache: